### DISCUSSION TITLE
Missing STL file

### DISCUSSION THREAD

**Discussion by TCorgard82** [sanni/cartreader#424] [Need help]
Hello so I am trying to build the newest revision. And the LCD bracket when I click on the link shows a preview but there is no download link. Would it be possible to acquire this file?

**Comment by VincentEmond** [comment]
Hey there, you can find the STL files here [https://github.com/sanni/cartreader/wiki/What-to-order#3d-prints](https://github.com/sanni/cartreader/wiki/What-to-order#3d-prints)

Click on download like shown on the picture below. 

![image](https://user-images.githubusercontent.com/9730015/180358379-4084acbe-7e73-4e80-a88b-9543d1a2acd5.png)

If it shows up as text instead of a download, just right click and Save As...

Hope this helps!

**Reply by TCorgard82** [reply]
this is for a different version, I was looking at this one, but I seen you can get the raw code, and if you put in notepad you can save as an stl.

https://github.com/sanni/cartreader/blob/214a6968bb1127d401387bcb47b3e6881fb0dbfc/hardware/six_slot_adapter/lcd_bracket.stl

**Reply by VincentEmond** [reply]
I see, in that case click on the link you gave me, navigate back to the cartreader main page using the breadcrumbs at the top of the page.
Then make sure the specific commit is selected like highlighted in yellow in the picture. Download the zip file, extract and navigate to `hardware/six_slot_adapter/lcd_bracket.stl`

![image](https://user-images.githubusercontent.com/9730015/180663980-169eb327-3a09-4e51-b017-d8680341ef94.png)
